app/views/sign_up/completions/_requested_attributes.html.erb

```diff
@@ -0,0 +1,13 @@
+<ul class="mb3 list-reset border-bottom success-bullets requested-attributes">
```

Comment:
I also did a slim-to-ERB conversion, so this looks like a new file but it's really just a translated one with a few small changes